### ISSUE TITLE
serve geobuf ?

### ISSUE THREAD

**Issue by AlenKelemen** [jumpinjackie/mapguide-rest#198]
Maybe it would be easy to implement  [geobuf](https://github.com/mapbox/geobuf) as third (and fast) option format.

**Comment by jumpinjackie** [comment]
2 problems:

 1. Geobuf is a protobuf-based format. This means it's a non-starter if I can't find library that can write protobuf messages. This library also has to be pure PHP as well as relying on native protobuf libraries means mapguide-rest loses its easy portability.
 2. Geobuf, like every other mapbox-based protobuf spec they is based on proto2 messages, which last I checked has weak to non-existent PHP tooling/codegen support. If it was a proto3-based spec, then at least I know I can tap into a wealth of codegen tools because they're all proto3 based.

So in summary, not very likely to happen.
